### PR TITLE
Sh empty folder

### LINKED ISSUE #223: Classifai screen stuck when import empty folder
**To Reproduce**
Import an empty folder

**Expected behavior**
Back to labelling tool without error

**Screenshots**
Capture

**Desktop (please complete the following information):**
 - Windows
- v1.0.0

### PR DESCRIPTION
**Pull request by toh-sie-hung**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #223

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Tested on?

- [x] Windows  
- [ ] Linux Ubuntu 
- [ ] Centos 
- [ ] Mac  
- [ ] Others  (State here -> xxx )  

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged

### CODE REVIEW

**Comment by toh-sie-hung** [comment]
Fixes the import empty folder problem. Once user import empty folder, the screen will back to labelling tool screen. Nothing new image will add and wont have message pop up. Able work on my laptop.

File: classifai-core/src/main/java/ai/classifai/util/data/ImageHandler.java [hunk 1]
```diff
@@ -320,8 +320,13 @@ public static void processFolder(@NonNull Integer projectID, @NonNull File rootP
                     totalFilelist.addAll(files);
                 }
             }
+
+            for(File f2 : totalFilelist)
+            {
```
Comment (codenamewei) [inline classifai-core/src/main/java/ai/classifai/util/data/ImageHandler.java:325]: need to remove the println command which is suppose for debugging @toh-sie-hung 
else in the long run the command prompt output will have random messages.

File: classifai-core/src/main/java/ai/classifai/selector/annotation/ToolFolderSelector.java [hunk 2]
```diff
@@ -92,11 +94,23 @@ protected JDialog createDialog(Component parent)
...
 
+                        String[] fileExtension = ImageFileType.getImageFileTypes();
+
                         if((rootFolder != null) && (rootFolder.exists()))
                         {
-                            loader.setFileSystemStatus(FileSystemStatus.WINDOW_CLOSE_LOADING_FILES);
+                            java.util.List<File> checkFileFormat = FileHandler.processFolder(rootFolder, fileExtension);
+
+                            if(checkFileFormat.isEmpty() != true)
+                            {
```
Comment (codenamewei) [inline classifai-core/src/main/java/ai/classifai/selector/annotation/ToolFolderSelector.java:104]: these operation happens in `ImageHandler.processFolder(projectID, rootFolder);` too

running it two times will cause the running time to massively increase, recommend to keep this file as how it was 

@toh-sie-hung

**Review by codenamewei — CHANGES_REQUESTED** [review]

**Comment by codenamewei** [comment]
Advise changing in [ImageHandler.java L 258](https://github.com/CertifaiAI/classifai/blob/sh_empty_folder/classifai-core/src/main/java/ai/classifai/util/data/ImageHandler.java#L258)

```
if(filesCollection.isEmpty())
        {
            loader.reset(FileSystemStatus.WINDOW_CLOSE_DATABASE_NOT_UPDATED);
            return;
        }
```

Screenshot 2020-11-19 223715

File: classifai-core/src/main/java/ai/classifai/util/data/ImageHandler.java (hunk 1 above)
Comment (toh-sie-hung) [reply classifai-core/src/main/java/ai/classifai/util/data/ImageHandler.java:325]: sorry, forget to remove it. Thx for checking

**Comment by toh-sie-hung** [comment]
okay, noted.

**Comment by toh-sie-hung** [comment]
I just moved the code from toolfolderselector file to imagehandle file

**Review by codenamewei — APPROVED** [review]
Works well now.

**Pull request merged**